src/Database/DBStructure.php

```diff
@@ -834,7 +834,7 @@ public static function checkInitialValues(bool $verbose = false)
 		}
 
 		if (self::existsTable('tag') && !DBA::exists('tag', ['id' => 0])) {
-			DBA::insert('tag', ['name' => '']);
+			DBA::insert('tag', ['name' => ''], Database::INSERT_IGNORE);
```

Comment:
I'm uneasy about this. It fixes the test error, but it doesn't address the part where there's a `tag = '' AND id != 0` row after a few cases of BBCode::convert tests, where tags shouldn't be inserted, and especially not with `tag = ''`.

Reply:
I wasn't not sure where to look at. I thought it was a timing issue that the rollback of the unit-test wasn't done and therefore it tried to override it again.

But tbh -  didn't investigate too much time because there are other issues on my list currently :-/